src/components/MonthCalendar.spec.tsx

```diff
@@ -32,12 +32,11 @@ describe('MonthCalendar', () => {
                 dummyCampaigns = data.data;
                 dummyCampaigns[0] = {
                     ...dummyCampaigns[0],
-                    'color': { bg: 'blue', fg: 'white' },
                     'id': 941,
                 };
                 dummyCampaigns[1] = {
                     ...dummyCampaigns[0],
-                    'color': { bg: 'red', fg: 'black' },
+                    'color': '#ff0000',
```

Comment:
Why not add this color to the fixture as well?

Reply:
the second campaign wasn't in the dummyCampaigns.json at all, i thought if i added the second campaign to the fixture, it might mess up some other tests? But theoretically it shouldn't, will add a second campaign to the array in fixture.

Reply:
Oh I understand, I didn't realize that's what you were doing but I understand it now. I don't think you need to change this if you don't really want to. 😊